app/controllers/users_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```

Comment:
remove this comment

Reply:
This @user call corresponds to the second form (form_tag). We commented it following the instructions from OdinProject